bedrock/firefox/templates/firefox/ios.html

```diff
@@ -122,6 +144,36 @@
   </div>{#-- /.container --#}
 </section>{#-- /#sync --#}
 
+{% if l10n_has_tag('tracking_protection') and waffle.switch('firefox-ios-global') %}
+<section class="section section-privacy" id="privacy">
+  <div class="container">
+    <h2 class="heading">{{ _('Keep your private info private') }}</h2>
+
+    <p class="lead">
+      {{ _('We believe your personal information belongs to you.') }}
+      {{ _('Firefox for iOS is filled with features to help keep it that way. ') }}
```

Comment:
Hm... I suspect that will break the string for l10n, but I'm not sure how strictly we deal with whitespace. @flodolo?